yarn-project/noir-contracts/src/contracts/inclusion_proofs_contract/src/utils.nr

```diff
@@ -0,0 +1,11 @@
+// Copied over from https://github.com/AztecProtocol/aztec-packages/blob/a07c4bd47313be6aa604a63f37857eb0136b41ba/yarn-project/noir-protocol-circuits/src/crates/rollup-lib/src/base/base_rollup_inputs.nr#L599
+// TODO: move to a shared place?
```

Comment:
Should these todos be addressed or made as separate issues?

Reply:
I didn't yet create issues for this because I felt like this needs discussion.

Would we consider it ok to import the protocol circuits? The historic access functionality will eventually most likely be moved to aztec-nr/aztec so we would have to add the dependency there.

@Maddiaa0 made similar comment [here](https://github.com/AztecProtocol/aztec-packages/pull/3255#discussion_r1407661638).

I think it at this point it would probably be ok to just have it as dependency and then eventually refactor these utils into a separate library if it becomes annoying to manage.

What do you guys think?

Reply:
I think we would end up having it in the aztec-nr, or very close to it at least.

Reply:
I created an issue and linked it in the code in [this commit](https://github.com/AztecProtocol/aztec-packages/pull/3255/commits/b8a1e6c168214ab3849616e42ddd95a6c1c983eb).

I think it makes sense to address this once both the protocol circuits and the historic access are more stable.